utils/utils.go

```diff
@@ -114,7 +68,11 @@ func GetEnvOrElse(name, orElse string) string {
 // IsTruthy returns true if a string is a truthy value.
 // Truthy values are "y", "yes", "true", "t", "on", "1" (case-insensitive); everything else is false.
 func IsTruthy(val string) bool {
-	switch strings.ToLower(strings.TrimSpace(val)) {
+	val = strings.TrimSpace(val)
+	if len(val) > 4 {
+		val = val[:4]
+	}
```

Comment:
doesn't this open up the possibility of accepting strings which are more than 4 characters but contain random trailing characters? eg: `true1234` would still pass since it is truncated at 4 characters.

Reply:
That's really true. I'll change to 5 and that will not be ambiguous.